legacy/setup-debian.sh

```diff
@@ -3,16 +3,14 @@
 export DEBIAN_FRONTEND=noninteractive
 apt-get update
 apt-get -y upgrade
-apt-get -y install autoconf automake binutils \
-	bison build-essential bzip2 ca-certificates coreutils \
-	curl debianutils file findutils flex g++ gcc gfortran git gzip \
-	hostname libbz2-dev libcurl4-openssl-dev libgsl-dev libicu-dev \
-	libfftw3-dev \
-	libgl1-mesa-dev libglu1-mesa-dev libgrpc++-dev \
-	liblzma-dev libncurses-dev libreadline-dev libsqlite3-dev libssl-dev libtool \
-	libx11-dev libxerces-c-dev libxext-dev libxft-dev \
-	libxml2-dev libxmu-dev libxpm-dev libyaml-cpp-dev libzstd-dev lsb-release make patch \
-	python3-dev protobuf-compiler-grpc rsync sed subversion tar unzip wget xutils-dev xz-utils
-apt-get -y -t buster-backports install cmake || \
-	apt-get -y install cmake
+apt-get -y install autoconf automake binutils bison build-essential bzip2 \
```

Comment:
I would propose to put only one package per line. In the diff one cannot find the one added package. With the proposed change this is true for this commit but will not be any problem in later changes.

Reply:
That's just a limitation of github :P 

Check git's `--word-diff` option:

![image](https://github.com/FairRootGroup/FairSoft/assets/297548/ce9e996c-6f5b-45d4-8f61-e2d34ebcd2cf)